packages/transaction-controller/src/utils/etherscan.ts

```diff
@@ -202,3 +194,21 @@ function getEtherscanApiUrl(
 
   return url;
 }
+
+/**
+ * Return the host url used to fetch data from Etherscan.
+ *
+ * @param chainId - Current chain ID used to determine subdomain and domain.
+ * @returns host URL to access Etherscan data.
+ */
+export function getEtherscanApiHost(chainId: Hex) {
```

Comment:
I appreciate this will be covered by the existing unit tests in `etherscan.test.test`, but should we a subset for this method directly to verify its usage independently?

Reply:
Added tests in `etherscan.test.test`